spec/registry.md

```diff
@@ -619,6 +619,24 @@ All other values produce an _Invalid Expression_ error.
 > or the type `com.ibm.icu.util.CurrencyAmount` can be used to set the currency and related
 > options (such as the number of fraction digits).
 
+### Digit Size Options
+
+Some _options_ of number _functions_ are defined to take a "digit size option".
+Implementations of number _functions_ use these _options_ to control aspects of numeric display
+such as the number of fraction, integer, or significant digits.
+
+A "digit size option" is an _option_ value that the _function_ interprets
+as a small integer value greater than or equal to zero.
+Implementations MAY define an upper limit on the resolved value 
+of a digit size option option consistent with that implementation's practical limits.
+
+In most cases, the value of a digit size option will be a string literal that
+encodes the value as a decimal integer.
+Implementations MAY also accept implementation-defined types as the value.
+The _literal_ representation of a digit size option matches the following ABNF:
```

Comment:
```suggestion
The textual representation of a digit size option matches the following ABNF:
```

This looks good except that using _literal_ here still puts one foot on each side of the abstraction barrier between function implementations and MessageFormat. I think what we really want to say is that a valid option is anything that resolves to a string matching `digit-size-option`. But I think "textual" gets it across.

Reply:
Well... what we really mean is "... anything that resolves to an integer between zero and some number that doesn't exceed 99..."

I used _literal_ here because I mean the _literal_ production in our syntax. Where literal doesn't belong is on line 633. One more fix...